src/Toxiproxy.hs

```diff
@@ -262,7 +262,7 @@ run = run' toxiproxyUrl
 run' :: BaseUrl -> ClientM a -> IO (Either ClientError a)
 run' url f = do
   manager <- newManager defaultManagerSettings
-  runClientM f (ClientEnv manager url Nothing)
+  runClientM f (mkClientEnv manager url)
```

Comment:
Were these changes in the upstream?

Reply:
Not relevant to this PR, but is there a description somewhere of why we are maintaining this?

Reply:
Well, it's in `tinypkgs`, so....
https://github.com/tinymce/tinypkgs/blob/master/CHANGELOG.org#added-toxiproxy-haskell-from-fork